src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java

```diff
@@ -2190,6 +2312,21 @@ public void createStreamControl(Map<String, String> stats, List<AdvancedControll
 		}
 	}
 
+	/**
+	 * This method is used to perform create stream control
+	 */
```

Comment:
Add throws ResourceNotReachableException in javadoc.

Reply:
I fixed it